crates/api_models/src/payments.rs

```diff
@@ -576,10 +576,18 @@ pub enum MandateReferenceId {
     NetworkMandateId(String), // network_txns_id send by Issuer to connector, Used for PG agnostic mandate txns
 }
 
-#[derive(Eq, PartialEq, Debug, serde::Deserialize, serde::Serialize, Clone)]
+#[derive(Debug, serde::Deserialize, serde::Serialize, Clone, Eq, PartialEq)]
```

Comment:
Try removing `Eq` and `PartialEq` just in case they're redundant. Comparing two potentially large data objects is not optimal.